bot.py

```diff
@@ -7,21 +7,24 @@
 from cogs.validation import ModeratorChecking
 from cogs.error_handler import CommandErrorHandler
 from cogs.help import GetHelp
+from cogs.member_stats import MemberStats
+# from .validation import is_moderator, is_mod_commands_channel
 
-bot = commands.Bot(command_prefix='$')
+intents = discord.Intents.default()
+intents.members = True
 
+bot = commands.Bot(command_prefix='$')
 
 @bot.event
 async def on_ready():
     print("Ready to go!")
 
-
 # Read in the token.
 with open(Path('./private/priv_data.json'), 'r') as data_file:
     token = json.loads(data_file.read())['TOKEN']
 
 bot.remove_command('help')  # Overwrite normal 'help' command
-#bot.add_cog(CommandErrorHandler(bot))
```

Comment:
This is my bad, because I had commented it out, but it should actually be kept in the code (although still commented out.) Sorry!